app/src/main/java/com/pokerio/app/screens/GameScreen.kt

```diff
@@ -62,9 +76,12 @@ fun GameScreen() {
         systemUiController.systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
 
         // Setup callbacks
-        val callbackId = GameState.addOnNewActionCallback {
-            debugNumberOfActions++
-        }
+        val newActionCallbackId =
+            GameState.addOnNewActionCallback {
```

Comment:
I think it is enough to just do `action++` in the callback, as the list and winnings pool would update as well as they use it as a key (just as in the lobby screen).

Reply:
Done.